modules/extensions-support/src/main/resources/META-INF/mule-extension-model.json

```diff
@@ -1825,6 +1825,561 @@
       ],
       "description": "Processes the nested list of message processors asynchronously using a thread pool",
       "modelProperties": {}
+    },
+    {
+      "name": "try",
+      "kind": "scope",
+      "blocking": true,
+      "executionType": "CPU_LITE",
```

Comment:
Just got confirmation this is ok, try should be BLOCKING and the rest are ok as CPU_LITE.